.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules/
 
 coverage/
+
+.idea/
```

Comment:
Small code style fix: IDE files should be in a developer’s local git ignore

https://docs.github.com/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

Reply:
Done! Thank you for the tip